docs/semgrep-sc/sc-getting-started.md

```diff
@@ -167,4 +167,27 @@ Semgrep Supply Chain enables developers to perform triage and remediation throug
 
 For more information, see [Triaging and remediating findings](/docs/semgrep-sc/triaging-and-remediating-vulnerabilities).
 
+## Appendix: Setting up SSC scans for specific project management tools
+
+### Apache Maven for Java
+
+Semgrep Supply Chain does not read `pom.xml` files to parse Maven projects. Instead it parses a dependency tree generated by `mvn`. Perform the following steps to enable Semgrep Supply Chain to correctly parse Maven projects:
+
+<ol>
+    <li>Generate a file outlining the project's dependency tree by adding the following command to your build pipeline:<br />
+    <pre class="language-bash">mvn dependency:tree -DoutputFile=maven_dep_tree.txt</pre>
+    For specific steps to add the command into your build pipeline, refer to your CI provider's documentation.</li>
+    <li>Run the Semgrep workflow, action, or step after the dependency tree has been generated.</li>
+</ol>
+
```

Comment:
@andy-r2c As far as I understand, a dependency tree MUST be generated per subdirectory? Is this only for monorepositories? 

Also related: how do we handle multi-file configurations?

Reply:
yeah, must be generated per subdirectory with a pom.xml whose contents the user wants scanned. More broadly for multi-file configurations (usually in monorepos), there's a few lines in our code that looks for the "closest" lockfile and associates the codefiles in that subdirectory with that, @mmcqd can probably share more details

Reply:
If a dependency tree is generated per subdirectory, the last thing I remember is that ssc will go from the "bottomost" file directory going upward towards root, parsing each folder that has a dependency tree. In short, it will bottom-up parse ALL the dependency tree files additively and the user doesn't have to do anything except generate those `maven_dep_tree.txt` files. Is this correct? @andy-r2c @mmcqd

Reply:
Do monorepos need to generate MANY maven_dep_tree.txt files is my question :)

Reply:
yeah @s-santillan exactly

Reply:
Okay, I added in an additional step in case they have a monorepo like structure! Resolving...